src/services/amqOnlineServices.js

```diff
@@ -139,24 +139,32 @@ const provisionAddressSpace = (username, namespace) => {
   return findOrCreateOpenshiftResource(addressSpaceDef(namespace), asRes);
 };
 
-const handleAddressSpaceUpdateEvent = (username, dispatch, resolve, event) => {
+const handleAddressSpaceUpdateEvent = (username, namespace, dispatch, resolve, event) => {
   if (!event || !event.status || !event.status.isReady) {
-    dispatch(getPayloadFromAddressSpace(event));
+    dispatch(getPayloadFromAddressSpace(event, username, namespace));
     return;
   }
   const messagingEndpointDef = event.status.endpointStatuses.find(e => e.name === 'messaging');
   if (!messagingEndpointDef) {
     return;
   }
-  const payload = getPayloadFromAddressSpace(event);
+  const payload = getPayloadFromAddressSpace(event, username, namespace);
   payload.additionalAttributes = Object.assign({}, payload.additionalAttributes, {
     'enmasse-credentials-username': cleanUsername(username),
     'enmasse-credentials-password': window.atob(genEvalPassword(username))
   });
   resolve(payload);
 };
 
-const getPayloadFromAddressSpace = as => {
+const buildAMQOnlineConcoleUrl = (username, namespace) => {
+  if (!window.OPENSHIFT_CONFIG.provisionedServices.amqonline) {
+    return null;
+  }
+  const baseUrl = `${window.OPENSHIFT_CONFIG.provisionedServices.amqonline.Host}`;
+  return `${baseUrl}/#/address-spaces/${namespace}/${username}/standard/addresses`;
```

Comment:
On this @pb82 . I would assume that if the address space is a brokered address space the end of this URL would be `brokered` rather than `standard`. I am not sure if the solution explorer ever creates a brokered addressspace though, so this might not be an issue

Reply:
Good point. I don't think the solution explorer can create a brokered address space, but i'll check if we can get the type of the address space somehow programmatically.

Reply:
perfect, thanks @pb82

Reply:
Updated now